src/stories/Library/pause-reservation/PauseReservation.tsx

```diff
@@ -0,0 +1,32 @@
+import ToggleButton from "../toggle-button/ToggleButton";
+
+export type PauseReservationProps = {
+  isChecked: boolean;
+};
+
+export const PauseReservation = ({ isChecked }: PauseReservationProps) => {
+  return (
+    <div className="dpl-pause-reservation-component">
+      <div className="dpl-pause-reservation-component__pagefold" />
+      <div className="dpl-pause-reservation-component__flex">
+        <div className="dpl-pause-reservation-component__flex__reservation-icon">
+          <img src="icons/collection/Reservations.svg" alt="Reservations" />
+        </div>
+        <div className="dpl-pause-reservation-component__flex__text">
+          Sæt fysiske reserveringer på pause
```

Comment:
I think you should make this controllable from the story, so the end user can experiment with the string